### ISSUE TITLE
Add hMailServer driver to "change password" plugin (or create standalone plugin)

### ISSUE THREAD

**Issue by setoy** [the-djmaze/snappymail#551]
There's currently no possibility to change an account's password when the underlying mailserver is [hMailServer](https://www.hmailserver.com/), which is one of the most popular ones for Windows.

Since the "change password" plugin does not support hMailServer yet (or won't ever because it's reliant on COM), one would possibly need a standalone plugin for that.

There [already exists one for RainLoop](https://github.com/RainLoop/rainloop-webmail/tree/master/plugins/hmailserver-change-password), but due to my limited ability in PHP, I failed to migrate it. I got it to the point where it showed up in the extensions manager, but no "Change Password" menu item appeared on the front end.

**Comment by the-djmaze** [comment]
You can try and test above added plugin.
Also available here: https://github.com/the-djmaze/snappymail/tree/master/plugins/change-password-hmailserver

**Comment by setoy** [comment]
Enabling this plugin results in the message `Use Change Password plugin` being displayed and the plugin not getting enabled. Also it tells me "Nothing to configure", but information for the COM account being used should be configured.

**Comment by the-djmaze** [comment]
Correct, you need both plugins.
Then you can configure hmailserver in the change password plugin.

This differs from RainLoop as now you can setup multiple drivers to change password, where RainLoop only supported one.

**Comment by setoy** [comment]
Okay so I need both downloaded but cannot enable the hMailServer one?

**Comment by the-djmaze** [comment]
1. Install `change password` plugin
2. Install this hmailserver plugin
3. Configure ` change password` plugin (has option for hmailserver now)

**Comment by setoy** [comment]
1:
Thanks for the clarification, this has worked. Should I enable the `Change Password hMailServer` extension after I have configured the `Change Password` extension? Because I could enable it now after configuring `Change Password`.

2:
I tried changing the password, but got the message "Password is too short", where it definitely wasn't. Might have something to do with the password complexity being too low? Because after using a shorter (but more complex) password, it worked.

Thanks!
